sdk/src/trace/tracer_provider.cc

```diff
@@ -66,14 +66,14 @@ nostd::shared_ptr<trace_api::Tracer> TracerProvider::GetTracer(
 
   for (auto &tracer : tracers_)
   {
-    auto &tracer_lib = tracer->GetInstrumentationLibrary();
+    auto &tracer_lib = tracer->GetInstrumentationScope();
```

Comment:
Just a side note (not related to this PR) - Also need to support [attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#get-a-tracer) added to InstrumentationScope as part of this PR - [Introduce Instrumentation Scope Attributes](https://github.com/open-telemetry/opentelemetry-specification/pull/2579). Will create a separate github issue for this.

Reply:
OK

Reply:
Can we add attributes in another PR.This PR just change the field name and use the new protobuf field to upload OTLP datas. So we can support the new system first.

Reply:
Yeah attributes should be separate PR. Will resolve this comment.
Thanks